api/config/genesis.go

```diff
@@ -11,6 +11,18 @@ type GenesisConfig struct {
 	Accounts map[string]uint64 `mapstructure:"accounts"`
 }
 
+// ToAccounts creates list of types.Account instance from config.
+func (g *GenesisConfig) ToAccounts() []types.Account {
+	var rst []types.Account
```

Comment:
```suggestion
	rst := make([]types.Account, 0, len(g.Accounts))
```
or it's not that big? :)

Reply:
it is probably not that big. but i tend to ignore preallocations when the code has no chance to have an impact on performance (e.g tests or node initialization)